src/types/Post.ts

```diff
@@ -0,0 +1,11 @@
+import { User } from './User';
+import { Comment } from './Comment';
+
+export interface Post {
+  userId: number;
+  id: number;
+  title: string;
+  body: string;
+  comments: Comment[],
+  user: User | null,
```

Comment:
For consistency, use a semicolon instead of a comma at the end of the line.

Reply:
also done